src/test/kotlin/uk/gov/justice/digital/hmpps/prisonestate/jpa/PrisonRepositoryTest.kt

```diff
@@ -46,6 +46,17 @@ class PrisonRepositoryTest {
   fun `should find associated gp practice`() {
     val prison = prisonRepository.findById("MDI").orElseThrow()
     assertThat(prison.gpPractice).isEqualTo(PrisonGpPractice("MDI", "Y05537"))
+  }
+
+  @Test
+  fun `should get active prisons`() {
+    val activePrisons = prisonRepository.findByActiveOrderByPrisonId(true)
+    assertThat(activePrisons).hasSizeGreaterThan(100)
+
+    val inActivePrisons = prisonRepository.findByActiveOrderByPrisonId(false)
+    assertThat(inActivePrisons).hasSizeGreaterThan(40)
```

Comment:
maybe also add `hasSizeLessThan(100)` to clarify you get less than the active ones

Reply:
I'll add
```kotlin
    assertThat(inActivePrisons).hasSizeGreaterThan(40).allMatch { !it.active }
```
into my next PR